try {
    // начать работу
    // работать
} catch (e) {
    // обработать ошибку
} finally {
    // очистить рабочее пространство
}

try {
    // начать работу
    // работать
} catch (e) {
    // обработать ошибку
}
// очистить рабочее пространство

//разница есть - например,при выходе из блока try сatch, при наличии return - finally гарантирует выполнение кода внутри
// код объявленный ниже в теле функции может просто не выполниться